test/parallel/test-http-localaddress.js

```diff
@@ -30,7 +30,8 @@ const assert = require('assert');
 
 const server = http.createServer((req, res) => {
   console.log(`Connect from: ${req.connection.remoteAddress}`);
-  assert.strictEqual(req.connection.remoteAddress, '127.0.0.2');
+  assert.match(req.connection.remoteAddress,
+               /^127\.0\.0\.1$|^127\.0\.0\.2$|^::1$/);
```

Comment:
Should the `127.0.0.1` address be accepted here? It wasn't before.
```suggestion
  assert.match(req.connection.remoteAddress, /^(127\.0\.0\.2|::1)$/);
```

Reply:
I'm not sure. ATM this is just to be less prone to failure and once everything passes, try to make it nice. Not sure what the original reason for `127.0.0.2` was anyways.